Service file for the filesystem monitor

iivanou opened the pull request:
Template of a service file in LSB format.

Commit:
Service file for the filesystem monitor

AndreiPashkin reviewed (changes requested):
Where is the file in LSB format?

iivanou commented:
The fsmonitor file is one you are looking for. This script supports all necessary actions. See below:
http://refspecs.linuxbase.org/LSB_3.0.0/LSB-PDA/LSB-PDA/iniscrptact.html 
A systemd service file will be created once we are ready to provide any kind of a packed application.

Commit:
Changes in file system monitor

1. Add test for copy(recursive) events
2. Add test for move events
3. We are hanlding only importnant events now
4. Replace move events with necessary ones
5. Reorginize tests for the monitor funtionality
6. Add necessary attributes to the Event class to match protobuf entity
7. Add new protobuf entities: is_dir, is_deleted
8. Generate new DTO for new entities

AndreiPashkin commented on entities/proto/onedrive_client/entities/common.proto:
```diff
@@ -16,6 +16,8 @@ import "onedrive_client/entities/onedrive.proto";
   uint64 modified = 2;
+  bool is_dir = 3;
+  bool is_deleted = 4;
 }
```
`is_deleted` definitely must be in `DirtyLocalItem`, because `DirtyLocalItem` reflects a change event (which includes deletion).
